manifests/check.pp

```diff
@@ -236,10 +238,53 @@
     publish             => $publish,
     dependencies        => $dependencies,
     custom              => $custom,
```

Comment:
Behavior has been moved to the Puppet DSL in the latest patch.  I'll proceed with it, let me know if you'd still like `content` removed after taking a look at the structure being built up.